Project.toml

```diff
@@ -8,6 +8,7 @@ ACE1x = "5cc4c08c-8782-4a30-af6d-550b302e9707"
 ACEbase = "14bae519-eb20-449c-a949-9c58ed33163e"
```

Comment:
> I also don't like this activation here. In general a user should create a working directory for their project, where ACEpotentials is added (to the local Project.toml). This will then be the activated project.